tests/go.mod

```diff
@@ -204,3 +204,7 @@ require (
 )
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+// Fix go mod tidy issue for ambiguous imports from go-ethereum
+// See https://github.com/ugorji/go/issues/279
+replace github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.1
```

Comment:
https://github.com/smartcontractkit/chainlink/tree/develop/integration-tests#common-issues

Reply:
I did not know about this fix. I have been doing the replace in multiple other libraries. I need to go fix those. Thanks for pointing this out to me!

Reply:
@jmank88 is the once who pointed it out to me today too!